clients/rospy/src/rospy/core.py

```diff
@@ -142,20 +143,37 @@ def rospyerr(msg, *args):
 def rospywarn(msg, *args):
     """Internal rospy client library warn logging"""
     _rospy_logger.warn(msg, *args)
-    
-logdebug = logging.getLogger('rosout').debug

```

Comment:
I think this is replaced in line 181 of the new file:

https://github.com/ros/ros_comm/pull/948/files#diff-84739190beb4f06aecd26a816f126e29R181